x/poolmanager/router_test.go

```diff
@@ -1896,11 +2049,12 @@ func (s *KeeperTestSuite) TestSplitRouteExactAmountIn() {
 			TokenInAmount: sdk.NewInt(twentyFiveBaseUnitsAmount.Int64() * 3),
 		}
 
-		priceImpactThreshold = sdk.NewInt(97469586)
+		priceImpactThreshold = sdk.NewInt(97866545)
```

Comment:
This value was not actually changed, just reverted to the value on main since default taker fee param was set to 0 in this PR